example-app/src/main/java/com/adyen/checkout/example/ui/configuration/CheckoutConfigurationProvider.kt

```diff
@@ -74,6 +74,7 @@ internal class CheckoutConfigurationProvider @Inject constructor(
     private fun getBcmcConfiguration(): BcmcConfiguration =
         BcmcConfiguration.Builder(shopperLocale, environment, clientKey)
             .setShopperReference(keyValueStorage.getShopperReference())
+            .setHolderNameRequired(true)
```

Comment:
Maybe let's not enable it by default in the example app, to be consistent with card